### PR TITLE
Add remote_read docs and update remote_write ones

### PR DESCRIPTION
**Pull request by juliusv**
Next step: add example code for remote read, point more precisely to
examples and specs for each.

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -995,16 +990,77 @@ Where `<scheme>` may be `http` or `https` and `<path>` is a valid URL path.
...
+url: <string>
+
+# Timeout for requests to the remote write endpoint.
+[ remote_timeout: <duration> | default = 30s ]
+
+# List of remote write relabel configurations.
+write_relabel_configs:
+  [ - <relabel_config> ... ]
+
+# Sets the `Authorization` header on every remote write request with the
```
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: Is there a better option to copy&pasting these everywhere?

Comment (juliusv) [inline content/docs/operating/configuration.md]: I was wondering this too, but the problem is that these are embedded fields in the config, and we'd have to make up another notation to describe that, making our config notation even more complex and confusing. So maybe just copy&paste is the least bad.

Comment (brian-brazil) [inline content/docs/operating/configuration.md]: As long as we can keep it updated everywhere, as this is going to end up in more files than this (AM, blackbox and consul will have them too).

**Comment by brian-brazil** [comment]
:+1:

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (juliusv) [inline content/docs/operating/configuration.md]: We can also consider making them non-embedded at some point, maybe for 2.0? Not sure if that is worth it though.

**Comment by grobie** [comment]
We could of course throw in even more ruby and generate our docs from some sets of definitions and embed them where needed. It'd make the source a lot harder to read though.

**Comment by juliusv** [comment]
@grobie Yeah, that way only we would have to deal with a special construct at least, not the user...

**Pull request merged**